js/main.js

```diff
@@ -225,12 +229,7 @@ app.on('activate', function () {
     }
 });
 
-// adds 'symphony' as a protocol in the system. plist file in macOS
-
-// on windows, we create the protocol handler via the installer
-// because electron leaves registry traces upon uninstallation
```

Comment:
I don't see `.aip` changes here so `.aip` will also add a registry entry?

Reply:
Yes, if there isn't an entry, it'll add.